src/Extensions/SiteTreeSubsites.php

```diff
@@ -106,7 +108,7 @@ public function onBeforeWrite()
     public function updateCMSFields(FieldList $fields)
     {
         $subsites = Subsite::accessible_sites('CMS_ACCESS_CMSMain');
-        $subsitesMap = [];
+        $subsitesMap = new Map(ArrayList::create());
```

Comment:
It may be better to put this into an else as we'd be using unnecessary memory to construct two needless objects here if there are subsites available.